server/src/main/java/com/exacaster/lighter/rest/SessionController.java

```diff
@@ -91,9 +93,9 @@ public Optional<Object> getLog(@PathVariable String id, @Nullable @Header("X-Com
     }
 
     @Post("/{id}/statements")
-    @Status(HttpStatus.CREATED)
-    public Statement postStatements(@PathVariable String id, @Valid @Body Statement statement) {
-        return sessionService.createStatement(id, statement);
+    public HttpResponse postStatements(@PathVariable String id, @Valid @Body Statement statement) {
```

Comment:
Since we've changed return type for this method, swagger will not know, what type of payload it returns, we should probably add something like:
```java
@ApiResponses(value = {
        @ApiResponse(responseCode = "201", description = "Statement created", content = { @Content(mediaType ="application/json", schema = @Schema(implementation = Statement.class)) }),
        @ApiResponse(responseCode = "400", description = "Invalid request"), 
        @ApiResponse(responseCode = "404", description = "Session not found")
})

Reply:
done